src/WebJobs.Extensions.DurableTask.Analyzers/Analyzers/Orchestrator/OrchestratorMethodCollector.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Analyzers
+{
+    public class OrchestratorMethodCollector
```

Comment:
The comments in your PR description were helpful in allowing me to understand what this method is for. Consider adding those comments in the code as `<summary>` information so that anyone who looks at the code can also get this same context.

Reply:
I tend to be pretty restrained with my comment usage. I forget that there are cases that comments are important. I added a few comments to some classes. Let me know if these seem helpful. Also let me know if you think adding comments would help explain other classes. Sometimes I can't see what isn't self-explanatory.